app/assets/javascripts/app/models/post/like_interactions.js

```diff
@@ -0,0 +1,57 @@
+// This class contains code extracted from interactions.js to factorize likes management between posts and comments
+
+app.models.Post.LikeInteractions = Backbone.Model.extend({
```

Comment:
Nitpick: Any reason to nest this into the `Post` namespace? Given this applies to comments as well now that feels a little weird to me.

Reply:
This Idea was inherited from Hank as initiator- what do you suggest?

Reply:
Just moving it to the top-level namespace perhaps?

Reply:
Sounds good. Will work in it in the next days.

Reply:
@jhass - ping? I've done some changes on the code as requested.